src/runtime_src/tools/scripts/pkgdsa.sh

```diff
@@ -479,6 +488,8 @@ initDsaBinEnvAndVars()
          fwManagement="${XILINX_XRT}/share/fw/mgmt.bin"
       elif [ "${CardMgmtControllerFamily}" == "CMC-Gen1" ]; then
          fwManagement="${XILINX_XRT}/share/fw/cmc.bin"
+      elif [ "${CardMgmtControllerFamily}" == "CMC-Gen2" ]; then
```

Comment:
@mfcpickett Can you work with the CMC FW developers to create a side name file like we do for BMC images.  Doing so will help avoid ALL of headaches when the binary changes.